.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea
 target
 /Talha.config
+pom.xml
```

Comment:
Hi, are you sure about adding pom.xml to gitginore?
Thnx!

Reply:
It is my idea and it will be better it stays not open to.